bson-kotlinx/src/main/kotlin/org/bson/codecs/kotlinx/BsonEncoder.kt

```diff
@@ -170,14 +171,7 @@ internal class DefaultBsonEncoder(
     override fun encodeDouble(value: Double) = writer.writeDouble(value)
     override fun encodeInt(value: Int) = writer.writeInt32(value)
     override fun encodeLong(value: Long) = writer.writeInt64(value)
-    override fun encodeNull() {
-        deferredElementName?.let {
-            if (configuration.explicitNulls) {
-                encodeName(it)
-            }
-        }
-        writer.writeNull()
-    }
+    override fun encodeNull() = writer.writeNull()
```

Comment:
Not sure how this was missed in a PR - but encoding null will never require the deferred element name - as encode(Nullable)SerializableValue will be used